example/exampleContour.html

```diff
@@ -68,56 +68,60 @@
       .mapboxToken(mapboxToken) // need a mapbox accessToken for loading the tiles
       .popupSearchRadius(2)
       .useGeoTypes(true) // need for projecting geo column using "mercator_map_projection"
-
+      
+      const intervalTypeFieldName = "is_major"
+      
       const contourPolygonLayer = dc.rasterLayer("polys")
       .crossfilter(crossFilter)
       .setState({
-          data: [
-            {
-              type: "contour",
-              table: config.table,
-              source: config.table,
-              contour_value_field: config.value,
-              lat_field: config.lat_field,
-              lon_field: config.lon_field,
-              agg_type: 'AVG',
-              fill_agg_type: 'AVG',
-              bin_dim_meters: config.bin_dim_meters,
-              neighborhood_fill_radius: 0,
-              fill_only_nulls: false,
-              flip_latitude: false,
-              minor_contour_interval: config.minor_contour_interval,
-              major_contour_interval: config.major_contour_interval,
-              contour_offset: 0,
-            }
-          ],
-          transform: {
-            projection: "mercator_map_projection",
-          },
-          mark: {
-            type: "poly",
-            strokeColor: "white",
-            strokeWidth: 0,
-            fillColor: "",
-            lineJoin: "miter",
-            miterLimit: 10
-          },
-          encoding: {
-            geocol: "contour_polygons",
-            color: {
-              type: "linear",
-              domain: [0, 6000],
-              range: ["#FF0000", "#0000FF"],
-              opacity: 0.50
-            }
-          },
-          enableHitTesting: false
-        })
+        data: [
+        {
+          type: "contour",
+          table: config.table,
+          source: config.table,
+          contour_value_field: config.value,
+          lat_field: config.lat_field,
+          lon_field: config.lon_field,
+          agg_type: 'AVG',
+          fill_agg_type: 'AVG',
+          bin_dim_meters: config.bin_dim_meters,
+          neighborhood_fill_radius: 0,
+          fill_only_nulls: false,
+          flip_latitude: false,
+          contour_offset: 0,
+          interval: {
```

Comment:
I personally like this better, tho it does bring into question whether some of the other properties should be grouped too.. i dunno. I think for typical uses defaults will be used, which would simplify this struct considerably.

Reply:
Yeah i'm not sure if any others squarely fit here as its currently named, unless we move all of the contour TF params into here or rename again. These are easy to change down the line, but let me know if you have strong feelings about moving some into this block or renaming things.